.github/workflows/test.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
     - name: Set up LXD
       run: |
-        sudo snap set lxd shiftfs.enable=true
+        sudo snap set lxd shiftfs.enable=true | { sleep 20 && sudo snap set lxd shiftfs.enable=true; }
```

Comment:
typo s/|/||/?

The autorefresh blocker seems to have done now anyway?

Reply:
😱

Reply:
Hah, and now I've committed to fix that, I can't approve!

Reply:
oh no, i can.merging!